pkgs/top-level/all-packages.nix

```diff
@@ -18981,6 +18981,8 @@ with pkgs;
 
   ### SCIENCE/MATH
 
+  almonds = callPackage ../applications/science/math/almonds { };
```

Comment:
Better use `pythonPackages.callPackage`. This way you won't need `pythonPackages` as an argument.